src/tidylinreg/tidylinreg.py

```diff
@@ -221,8 +237,12 @@ def get_pvalues(self):
         >>> model.fit(y, X)
         >>> model.get_pvalues()
         '''
-        return
-    
+        self.df = self.n_samples - self.n_features
```

Comment:
Should the degrees of freedom be n_samples - n_features - 1? Since we also have to include the intercept:
```suggestion
        self.df = self.n_samples - self.n_features - 1
```